RecoTracker/DisplacedRegionalTracking/interface/DisplacedVertexCluster.h

```diff
@@ -0,0 +1,66 @@
+#ifndef RecoTracker_DisplacedRegionalTracking_DisplacedVertexCluster_h
+#define RecoTracker_DisplacedRegionalTracking_DisplacedVertexCluster_h
```

Comment:
Is this class foreseen to be used outside of `DisplacedRegionSeedingVertexProducer`? If not, it would be better to move this header and the corresponding source file to the `plugins` subdirectory.

Reply:
It should only be used by the plugin, so I moved it into the plugins/ directory.